watcher/src/bin/main.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+#![feature(external_doc)]
+#![doc(include = "../../README.md")]
+
+use mc_watcher::{config::WatcherConfig, watcher::Watcher, watcher_db::create_or_open_watcher_db};
+
+use mc_common::logger::{create_app_logger, o};
+use mc_ledger_sync::ReqwestTransactionsFetcher;
+use structopt::StructOpt;
+
+fn main() {
+    mc_common::setup_panic_handler();
+    let (logger, _global_logger_guard) = create_app_logger(o!());
+
+    let config = WatcherConfig::from_args();
+
+    let transactions_fetcher =
+        ReqwestTransactionsFetcher::new(config.tx_source_urls.clone(), logger.clone())
+            .expect("Failed creating ReqwestTransactionsFetcher");
+
+    let watcher_db = create_or_open_watcher_db(
+        config.watcher_db,
+        &transactions_fetcher.source_urls,
+        logger.clone(),
+    )
+    .expect("Could not create or open watcher db");
+    let watcher = Watcher::new(watcher_db, transactions_fetcher, logger);
+    // For now, ignore origin block, as it does not have a signature.
+    watcher
```

Comment:
Shouldn't this be in a loop?

Reply:
This main just syncs and stops. The loop management is done by the WatcherSyncThread, which is currently only used in mobilecoind. If we decide to start shipping a watcher product, we can adjust this main to use the WatcherSync, but for now this is a test utility to test the watcher in isolation from mobilecoind.

I'll update the documentation to reflect that this tool syncs signatures until it finds no more blocks and then stops.